flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -557,6 +557,16 @@ private void createDB() throws IOException {
 		this.defaultColumnFamily = columnFamilyHandles.get(0);
 	}
 
+	private static class Utils {
```

Comment:
Why is the static class required, couln't we just have a private method?

Reply:
valid point, I initially thought to create utility  class, then decided to keep it local.